authors/detail.php

```diff
@@ -0,0 +1,60 @@
+<?php
+include '../csv_util.php';
+$line_counter = 0;
+$index = $_GET['index'];
+$authors_array = read_csv('../data/authors.csv');
+$author_name = $authors_array[$index];
+
+function printQuote($filename,$index){
+    $fh=fopen('../data/quotes.csv', 'r');
+
+    $i=0;
+    while($line=fgets($fh)) {
+    $line_array = explode(";",$line);
+    if ($index==trim($line_array[0])) {
+        echo "\"".$line_array[1]."\"<br>";
+    }
+    $i = $i + 1;
+    }
+    fclose($fh);
+    }
```

Comment:
Do you think this could benefit from a function from our `csv_util.php` class?

Reply:
function on csv_util will change the implementation of this function